src/main/java/org/kiwiproject/dropwizard/error/ErrorContextUtilities.java

```diff
@@ -75,4 +78,24 @@ static RecentErrorsHealthCheck registerRecentErrorsHealthCheckOrNull(boolean add
 
         return null;
     }
+
+    static CleanupApplicationErrorsJob registerCleanupJobOrNull(boolean addCleanupJob,
```

Comment:
Is there any reason to store the actual job in the `ErrorContext` the way we do with the health check? i.e. add a method to the `ErrorContext` interface:

```java
Optional<CleanupApplicationErrorsJob> cleanupApplicationErrorsJob();
```

At present there is not much a caller could do with the `CleanupApplicationErrorsJob` other than `run()` it manually, which is probably not something we'd want them doing manually.

But perhaps if we added methods to obtain the `CleanupConfig` or maybe if we added some kind of internal stats for deletions over time, it might make more sense. Thoughts?

Reply:
I followed that path at first when I was adding this but decided that we didn't need it stored for now.  If we ever find a need to access something from the job we can always add that in pretty easily later.